.github/project-root-duplicates/build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+plugins {
```

Comment:
in the future, it will be better if we can automate the creation of the duplicates so that it doesn't require another 'out of sync' check. just a thought for future improvement.